sdk/keyvault/keyvault-admin/README.md

```diff
@@ -34,16 +33,151 @@ TypeScript users need to have Node type definitions installed:
+    ```
+    Output:
+    ```
+    "<your-service-principal-object-id>"
+    ```
+* Use the returned credentials above to set  **AZURE_CLIENT_ID** (appId), **AZURE_CLIENT_SECRET** (password), and **AZURE_TENANT_ID** (tenant) 
+environment variables. For example, you can use the [dotenv][dotenv] project and add a `.env` file to your project with hte following contents (for the full section of environment variables, see [Environment Variables](#environment-variables)):
+    ```PowerShell
+    AZURE_TENANT_ID=<tenant-ID>
+    AZURE_CLIENT_ID=<generated-app-ID>
+    AZURE_CLIENT_SECRET=<random-password>
+    ```
+
+* Create the Key Vault and grant the above mentioned application authorization to perform administrative operations on the Azure Key Vault 
+(replace `<your-resource-group-name>` and `<your-key-vault-name>` with your own, unique names and `<your-service-principal-object-id>` with the value from above):
+    ```
+    az keyvault create --hsm-name <your-key-vault-name> --resource-group <your-resource-group-name> --administrators <your-service-principal-object-id> --location <your-azure-location>
+    ```
+
+* Use the above mentioned Azure Key Vault name to retrieve details of your Vault which also contains your Azure Key Vault URL:
+    ```PowerShell
+    az keyvault show --hsm-name <your-key-vault-name>
+    ```
+
+#### Get or create a storage account
+
+To generate Key Vault backups, you will need to point the [KeyVaultBackupClient][src-backup-client] to an existing Storage account.
+
+To create a new Storage Account, you can use the [Azure Portal][storage-account-create-portal],
+[Azure PowerShell][storage-account-create-ps], or the [Azure CLI][storage-account-create-cli].
+Here's an example using the Azure CLI:
+
+```Powershell
+az storage account create --name MyStorageAccount --resource-group MyResourceGroup --location westus --sku Standard_LRS
+```
+
+#### Environment Variables
+
+With a Key Vault and a Storage account, you can fill the rest of the environment variables in the `.env` file used by [dotenv][dotenv]. You can look for more information about other required environment variables in our [sample.env](./sample.env) file.
+
+#### Create KeyVaultAccessControlClient
+
+Once you've populated the **AZURE_CLIENT_ID**, **AZURE_CLIENT_SECRET** and **AZURE_TENANT_ID** environment variables and replaced **your-vault-url** 
+with the above returned URI, you can create the [KeyVaultAccessControlClient][src-rbac-client]:
+
+```ts
+const credentials = new DefaultAzureCredential();
+const client = new KeyVaultAccessControlClient(vaultUrl, credentials);
+```
+
+#### Create KeyVaultBackupClient
 
-### Configuring your Key Vault
+Once you've populated the **AZURE_CLIENT_ID**, **AZURE_CLIENT_SECRET** and **AZURE_TENANT_ID** environment variables and replaced **your-vault-url** 
+with the above returned URI, you can create the [KeyVaultBackupClient][src-backup-client]:
+
+```ts
+const vaultUrl = `https://<MY KEY VAULT HERE>.vault.azure.net`;
```

Comment:
nit: this snippet has this extra `vaultUrl` line compared to the previous snippet, can you also include this line in the previous snippet?